src/Command/Autoscaling/AutoscalingSettingsSetCommand.php

```diff
@@ -0,0 +1,776 @@
+<?php
+
+namespace Platformsh\Cli\Command\Autoscaling;
+
+use Platformsh\Cli\Command\CommandBase;
+use Platformsh\Client\Exception\EnvironmentStateException;
+use Platformsh\Client\Model\AutoscalingSettings;
+use Platformsh\Client\Model\Deployment\Service;
+use Platformsh\Client\Model\Deployment\WebApp;
+use Platformsh\Client\Model\Deployment\Worker;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class AutoscalingSettingsSetCommand extends CommandBase
+{
+    protected function configure()
+    {
+        $this->setName('autoscaling:set')
+            ->setDescription('Set the autoscaling configuration of apps or workers in an environment')
+            ->addOption('service', '-s', InputOption::VALUE_REQUIRED, 'Name of the app or worker to configure autoscaling for')
```

Comment:
```suggestion
            ->addOption('service', 's', InputOption::VALUE_REQUIRED, 'Name of the app or worker to configure autoscaling for')
```

Reply:
Thanks, again something about php that was not obvious to me. I thought it meant the flag was optional.